### PR TITLE
fix ''dict object has no attribute dict_keys" issue with python3

### PR DESCRIPTION
**Pull request by jmlrt**
In Python2, the dict.keys(), dict.values(), and dict.items() methods returns a list. Jinja2 returns that to Ansible via a string representation that Ansible can turn back into a list. In Python3, those methods return a dictionary view object.
resource: https://docs.ansible.com/ansible/2.4/playbooks_python_version.html#dictionary-views

Fix https://github.com/elastic/ansible-elasticsearch/issues/570

### CODE REVIEW

File: tasks/xpack/security/elasticsearch-security-file.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 - name: set fact manage_file_users
-  set_fact: manage_file_users=es_users is defined and es_users.file is defined and es_users.file.keys() | length > 0
```
Comment (Crazybus) [inline tasks/xpack/security/elasticsearch-security-file.yml]: Why are some of these now `.keys() | list` while others are just `list` with the `.keys()` also removed? I can't figure out why some still need `.keys()` and some don't.

File: tasks/xpack/security/elasticsearch-security-file.yml [hunk 2]
```diff
@@ -72,7 +72,7 @@
   become: yes
   command: >
     {{es_home}}/bin/{{es_xpack_users_command}} passwd {{ item }} -p {{es_users.file[item].password}}
-  with_items: "{{ es_users.file.keys() | default([]) }}"
```
Comment (Crazybus) [inline tasks/xpack/security/elasticsearch-security-file.yml]: Can the `default([])` be removed here? Does list automatically return an empty list if the variable is undefined? 

It fails for me on https://ansible.sivel.net/test/ but maybe it doesn't handle empty variables properly.

File: tasks/xpack/security/elasticsearch-security-file.yml (hunk 1 above)
Comment (jmlrt) [reply tasks/xpack/security/elasticsearch-security-file.yml]: just a mistake => fixed

File: tasks/xpack/security/elasticsearch-security-file.yml (hunk 2 above)
Comment (jmlrt) [reply tasks/xpack/security/elasticsearch-security-file.yml]: With the `when: manage_file_users` and `set_fact: manage_file_users=es_users is defined and es_users.file is defined and es_users.file.keys() | list | length > 0` at the top of the file, if the variable is undefined we will skip this task so I think we can avoid having an additional `default` filter here, WDYT.

File: tasks/xpack/security/elasticsearch-security-file.yml (hunk 1 above)
Comment (Crazybus) [reply tasks/xpack/security/elasticsearch-security-file.yml]: There were a few more were it was removed. Which is why I thought it was on purpose. At least 2 more in this PR where it was deleted.

File: tasks/xpack/security/elasticsearch-security-file.yml (hunk 2 above)
Comment (Crazybus) [reply tasks/xpack/security/elasticsearch-security-file.yml]: Makes sense to me!

File: tasks/xpack/security/elasticsearch-security-file.yml (hunk 1 above)
Comment (jmlrt) [reply tasks/xpack/security/elasticsearch-security-file.yml]: Are you sure? I don't see no more `| list` without `.keys()`

**Comment by svanschalkwyk** [comment]
Thanks! Quick turn-around!

**Pull request merged**